src/sidebar/components/annotation-action-bar.js

```diff
@@ -44,7 +44,7 @@ function AnnotationActionBar({
   const onDelete = () => {
     if (window.confirm('Are you sure you want to delete this annotation?')) {
       annotationsService.delete(annotation).catch(err => {
-        toastMessenger.error(err.message, 'Deleting annotation failed');
```

Comment:
The second parameter here was unused previously, and is now used for an options object that contains additional per-message display options.